includes/Classifai/Services/ServicesManager.php

```diff
@@ -343,4 +345,31 @@ public function check_license_key( $email, $license_key ) {
 
 		return false;
 	}
+
+	/**
+	 * Adds debug information to the ClassifAI Site Health screen.
+	 *
+	 * @param array      $debug_information Array of lines representing debug information.
+	 * @param array|null $settings Settings array. If empty, will be fetched.
+	 * @return array Array with lines added.
+	 * @since 1.4.0
+	 */
+	public function add_debug_information( $debug_information, $settings = null ) {
+		if ( is_null( $settings ) ) {
+			$settings = $this->sanitize_settings( $this->get_settings() );
+		}
+
+		$valid_licence       = 1 === intval( $settings['valid_license'] ?? 0 );
+		$debug_information[] = [
+			'label' => __( 'Valid license', 'classifai' ),
+			'value' => $valid_licence,
+		];
+
+		$debug_information[] = [
+			'label' => __( 'Email', 'classifai' ),
+			'value' => $settings['email'] ?? '',
```

Comment:
I think we ran into an issue using the null coalescing operator with a client install - @helen @jeffpaul ?

Reply:
I'm interested in the answer to this question and will hold on my updates until we have it.

FWIW, I took the 2-3 existing instances of the null coalescing operator as implicit permission -- e.g.: 

https://github.com/10up/classifai/blob/develop/includes/Classifai/Providers/Watson/NLU.php#L298

Of course, switching them all out with the PHP5-friendly equivalents is no problem.

Reply:
@johnwatkins0 I followed up on this and this was the issue I was thinking of #78. I think we're fine to move forward with the null coalescing operator. Thanks for your patience!